plugins/org.eclipse.embedcdt.debug.gdbjtag.qemu.core/src/org/eclipse/embedcdt/debug/gdbjtag/qemu/core/dsf/DebuggerCommands.java

```diff
@@ -190,5 +190,11 @@ public IStatus addStartRestartCommands(boolean doReset, List<String> commandsLis
 		return Status.OK_STATUS;
 	}
 
+	@Override
+	public IStatus addMstatusCSRResetCommands(List<String> commandsList) {
```

Comment:
Is this absolutely necessary?

Reply:
Yes, because this method is defined in the parent interface (IGnuMcuDebuggerCommandsService)